client/src/modules/Score/components/ScoreTable/index.tsx

```diff
@@ -161,6 +161,24 @@ export function ScoreTable(props: Props) {
     return null;
   }
 
+  function trimFiltersProps(filters: ScoreTableFiltersModified): ScoreTableFiltersModified {
+    const trimmedFiltersProps = structuredClone(filters);
+    const filtersKeys = Object.keys(filters) as (keyof ScoreTableFiltersModified)[];
+
+    filtersKeys.forEach(key => {
+      const filterKey = filters[key];
+      if (filterKey && typeof filterKey !== 'boolean') {
+        key = key as Exclude<keyof ScoreTableFiltersModified, 'activeOnly'>;
+        if (typeof filterKey === 'object') {
```

Comment:
`Array.isArray()` seems more appropriate to use here

Reply:
ok I did it.